integration_test/third_party_apps_data/applications/mysql/expected_metrics.yaml

```diff
@@ -1,21 +1,37 @@
+- type: workload.googleapis.com/mysql.buffer_pool_data_pages
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    status: dirty|flushed
+- type: workload.googleapis.com/mysql.buffer_pool_limit
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels: {}
 - type: workload.googleapis.com/mysql.buffer_pool_operations
   value_type: INT64
   kind: CUMULATIVE
   monitored_resource: gce_instance
   labels:
     operation: read_ahead|read_ahead_evicted|read_ahead_rnd|read_requests|reads|wait_free|write_requests
+- type: workload.googleapis.com/mysql.buffer_pool_page_flushes
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels: {}
 - type: workload.googleapis.com/mysql.buffer_pool_pages
   value_type: DOUBLE
   kind: GAUGE
   monitored_resource: gce_instance
   labels:
-    kind: data|dirty|flushed|free|misc|total
+    kind: data|free|misc
 - type: workload.googleapis.com/mysql.buffer_pool_size
   value_type: DOUBLE
   kind: GAUGE
   monitored_resource: gce_instance
   labels:
-    kind: data|dirty|total
+    kind: data|dirty
 - type: workload.googleapis.com/mysql.commands
```

Comment:
If this is the PR that pulls in the fixed command labels then we should add them here too. If I've been following correctly, it should contain the values from `agent.googleapis.com/mysql/command_count` in the discrepancy doc (i.e. the super long list of like 100 different values). cc @igorpeshansky to confirm

Reply:
I don't believe that those changes have yet been made upstream, but will wait for Igor's confirmation.

Reply:
I recall there was a conversation and the conclusion is that `total` is a buggy label and mess up the chart aggregation. So the decision is to treat it as a bug fix and remove `total`

Reply:
Confirmed with Igor offline. We can remove `total`.

Reply:
The `commands` metric has not yet been fixed — it still only pulls in prepared statement metrics.